CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.19.0
```

Comment:
List looks good, couple nitpicks re: consistency w/ previous changelogs:
* Let's put features at the top since they're the biggest deal
* Can we phrase features as "Add support for..."
* Capitalize descriptions